daprdocs/content/en/operations/components/setup-bindings/supported-bindings/_index.md

```diff
@@ -12,6 +12,7 @@ Every binding has its own unique set of properties. Click the name link to see t
 
 | Name | Input<br>Binding | Output<br>Binding | Status |
 |------|:----------------:|:-----------------:|--------|
+| [Alibaba Cloud OSSs]({{< ref alicloudoss.md >}})           |    | ✅ | Alpha |
```

Comment:
Should we split this out into an Alibaba cloud section similar to Amazon/Azure?

Reply:
Split it.